remote-urls.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+urls:
+  ramble: https://github.com/GoogleCloudPlatform/ramble.git
+  spack: https://github.com/spack/spack.git
```

Comment:
(question) this seems like it would be useful but I can't immediately think of how - did you have something in mind for this? (I'm fine with it going in either way)

Reply:
I think this is a preferable interface if the user wants to change these to point at a different than hardcoding the urls in `lib/`.